packages/fal/src/fal-image-model.ts

```diff
@@ -83,6 +87,19 @@ export class FalImageModel implements ImageModelV2 {
     const downloadedImages = await Promise.all(
       targetImages.map(image => this.downloadImage(image.url, abortSignal)),
     );
+    const {
+      // @ts-expect-error - either image or images is present, not both.
```

Comment:
is there a typesafe way to handle this?

Reply:
I'm not using `image` or `images`, the code is just to filter keys fro `responseMetaData`. It's the most efficient way as far as I know.